### ISSUE TITLE
iceprod2 in cvmfs (Trac #1107)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1101]
Add support for iceprod2:

regular programs:

* p7zip
* nginx

python packages:

* apsw
* futures
* pycurl
* setproctitle

Anyone against just putting these in the main icecube repo?

Migrated from https://code.icecube.wisc.edu/ticket/1107
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Add support for iceprod2:\n\nregular programs:\n\n* p7zip\n* nginx\n\npython packages:\n\n* apsw\n* futures\n* pycurl\n* setproctitle\n\nAnyone against just putting these in the main icecube repo?", 
    "reporter": "david.schultz", 
    "cc": "nega, olivas", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "cvmfs", 
    "summary": "iceprod2 in cvmfs", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-11T19:23:34", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T19:06:08: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150922T18:16:59: david.schultz commented:

> poke for nega and olivas.  a yea or nay would be nice.

**Comment by icecube-trac** [comment]
Trac update at 20151125T17:29:46: david.schultz changed priority from "critical" to "blocker"

**Comment by icecube-trac** [comment]
Trac update at 20160105T22:12:19: david.schultz changed description from:

> Add support for iceprod2:
> 
> regular programs:
> 
> * p7zip
> * nginx
> 
> python packages:
> 
> * apsw
> * setproctitle
> 
> Anyone against just putting these in the main icecube repo?

to:

> Add support for iceprod2:
> 
> regular programs:
> 
> * p7zip
> * nginx
> 
> python packages:
> 
> * apsw
> * futures
> * pycurl
> * setproctitle
> 
> Anyone against just putting these in the main icecube repo?

**Comment by icecube-trac** [comment]
Trac update at 20160106T22:02:04: olivas commented:

> Why do you want these in the main repo?

**Comment by icecube-trac** [comment]
Trac update at 20160106T22:04:49: david.schultz commented:

> Mostly because it's easy.  I can make another repo too.  Is that better / worth the effort?  I'm not sure.

**Comment by icecube-trac** [comment]
Trac update at 20160106T22:12:39: olivas commented:

> If it's read-only and only contains releases I'm not opposed, but I'm going to defer to nega on this one.  By "main repo," you mean our svn repo, right? Not our WIPAC github repo...or our other public github repo?

**Comment by icecube-trac** [comment]
Trac update at 20160106T22:23:19: david.schultz commented:

> Ok, I guess I need to clarify terms:
> 
> In svn:
> 
> * IceProd2 source:
>   * currently at source:IceCube/sandbox/dschultz/iceprod2
>   * IceProd v1 is at source:IceCube/meta-projects/iceprod
>   * not sure if we really want to call IceProd2 a meta-project anymore
>   * we could also move it to github if someone felt like it
> 
> In CVMFS:
> 
> * IceCube main software:
>   * py2-v1
>   * py2-v2
> 
> * either add the IceProd2 software (what this ticket is referring to) to those, or make a separate one:
>   * iceprod-v2.0.0
> 
> 
>

**Comment by icecube-trac** [comment]
Trac update at 20160113T23:10:57: david.schultz commented:

> talked a bit on slack.  going with option 2, making a separate iceprod section in cvmfs.

**Comment by icecube-trac** [comment]
Trac update at 20160120T20:56:05: 
* david.schultz commented:

> In changeset 141078:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141078"
> some fixes for py2-v2, to stay working. add iceprod 2.0.0 repo. closes #1101
> ```


* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
